addons/website/models/ir_translation.py

```diff
@@ -35,11 +35,7 @@ def load_module_terms(self, modules, langs):
                 ON t.type = 'model' AND t.name = 'website.menu,name' AND t.res_id = o_menu.id
              INNER JOIN website_menu s_menu
                 ON o_menu.name = s_menu.name AND o_menu.url = s_menu.url
-             INNER JOIN website_menu root_menu
-                ON s_menu.parent_id = root_menu.id AND root_menu.parent_id IS NULL
-             WHERE t.lang IN %s and t.module IN %s
-               AND o_menu.website_id IS NULL AND o_menu.parent_id = %s
-               AND s_menu.website_id IS NOT NULL""" + conflict_clause,
-            (tuple(langs), tuple(modules), default_menu.id))
```

Comment:
```sql
AND o_menu.website_id IS NULL AND s_menu.website_id IS NOT NULL""" + conflict_clause,
```

Get ride off `AND o_menu.parent_id = %s` only
because you got values not expected 

Look you are getting 3 rows for services and About
![image](https://user-images.githubusercontent.com/10233975/81459298-d684b580-9164-11ea-9fbf-96d6b151f609.png)

Reply:
In this base, with dummy_module it creates 2 Websites and the Undefined one, I'm not really sure why for the Apps, it's only being created on the websites

Reply:
![Task 39008 - Websites](https://user-images.githubusercontent.com/26889951/81460942-2ae06300-916e-11ea-9cf3-6c6af2a2e3da.png)

If I add the conditions `AND o_menu.website_id IS NULL AND s_menu.website_id IS NOT NULL` I got the following result:
![Task 39008 - SQL sentence with conditions](https://user-images.githubusercontent.com/26889951/81460961-5400f380-916e-11ea-91a3-07c2c8d38026.png)

Reply:
We only let the condition `AND s_menu.website_id IS NOT NULL`